contracts/contracts/metrics/Statistics.sol

```diff
@@ -31,6 +30,8 @@ contract Statistics is Initializable {
     mapping(address => mapping(address => LevelInstance)) private playerStats;
     mapping(address => bool) private playerExists;
     mapping(address => bool) private levelExists;
+    mapping(address => uint256) private averageTimeTakenToCompleteLevels;
```

Comment:
Because of this variable we should change from this:

```
    uint256[45] private __gap;

```

To 

```
    uint256[44] private __gap;

```